README.md

```diff
@@ -1,6 +1,5 @@
-WebVR Community Group
-=====================
+# WebVR Specification
 
-This is the repository for the [WebVR Community Group](https://www.w3.org/community/webvr/).
+This repository houses the [WebVR specification](https://mozvr.github.io/webvr-spec/), in [Bikeshed](https://github.com/tabatkins/bikeshed) format.
```

Comment:
https://mozvr.github.io/webvr-spec/ should be https://w3c.github.io/webvr/ after the migration.